stemcell_builder/lib/prelude_apply.bash

```
@@ -59,9 +59,8 @@ function update_kernel_static_libraries {
   sed -i "s/__KERNEL_VERSION__/$kernel_version/g" $chroot/var/vcap/bosh/etc/static_libraries_list
 
   # since kernel 6.5 the objtool/libsubcmd.a has been moved to objtool/libsubcmd/libsubcmd.a
-  # fips still uses the 5.9 kernel, so we need to rewrite the static_libraries_list file to the old path
-  # TODO: puting a todo tag here as this could probably be removed in the future if the kernel version gets bumped
-  if [[ "${stemcell_operating_system_variant}" == 'fips' ]]; then
+  # older fips kernels (5.x) used the old path, so rewrite only for those
+  if [[ "${stemcell_operating_system_variant}" == 'fips' ]] && [[ "${major_kernel_version}" < "6.0" ]]; then
```

Shellcheck (0.11.0)

[warning] stemcell_operating_system_variant is referenced but not assigned.

(SC2154)